profiles/prefix/package.mask

```diff
@@ -8,6 +8,19 @@
 # NOTE: Please add your entry at the top!
 #
 
+# For testing purpose
+sys-auth/openpam
+sys-auth/pam_abl
+sys-auth/pam_fprint
+sys-auth/pam_mktemp
+sys-auth/pam_mount
+sys-auth/pam_mysql
+sys-auth/pam_passwdqc
+sys-auth/pam_skey
+sys-fs/udev
+sys-libs/pam
+virtual/pam
```

Comment:
We'd better modify `virtual/pam` so that it has a dummy dependency on Prefix.  Otherwise, a lot of ebuilds depending on `virtual/pam` will need to be updated.